contributing/content-style-guide.md

```diff
@@ -279,6 +279,43 @@ A table following an H3 heading:
 
 ## Word choice & terminology
 
+### Active voice
+
+Use the active voice most of the time. Use the passive voice sparingly. 
+
+- Passive voice: _Files are added by developers._
+- Active voice: _Developers add files._
+
+Active voice provides the following advantages:
+
+- Active voice is generally shorter than passive voice.
+- Active voice is easier for users to understand and often results in shorter content.
+- Most readers mentally convert passive voice to active voice. Why subjects readers to extra processing time? By sticking to active voice, you enable readers to skip the preprocessor stage and go straight to compilation.
+- Passive voice confuses your ideas and reports action indirectly.
+- Some passive voice sentences omit an actor altogether, which forces the reader to guess the actor's identity.
+
+
+Sometimes, using passive voice is appropriate. Make sure it’s an intentional choice that communicates the idea clearer than active voice would. For example, when the system is the actor, rather than a person.
+
+
+✅ Use |	❌ Avoid
+--- | ---|
+(Active voice) Ask someone with access to dbt Cloud to transform the data. | This data transformation can be done by someone with access to dbt Cloud. |
+(Passive voice &mdash; exceptions) The open-sourced rpc plugin is used by the Cloud IDE to recompile changes made in your project. |  We are using the open-sourced rpc plugin for the Cloud IDE to recompile changes made in your project. |
+
+### Spelling
+
+In general, when the spelling of English words varies by locale &mdash; use the US spelling. For example:
+
+✅ Use | ❌ Avoid
+-- | --
+standardize  | standardise 
+license | licence
+color | colour
+</div></b>
+
+Although non-US readers may find it difficult to have to type US spelling. For example, if your documentation tells a reader who's used to spelling `colour` to type `color`, they may mistype it. So when you use filenames or data parameters in examples, do your best to avoid words that are spelled differently by different groups of English speakers.
```

Comment:
```suggestion
Avoid regional idiomatic phrases as well.  For example, a common saying amongst English speakers in India is "do the needful," but this phrase may be unrecognizable to English speakers from other regions. 
```

Reply:
When addressing people internationally, one best practice is to avoid referring to them as "non-US" (or "non-English" speakers, as I've done in the past). It centers on the US and can indicate to readers that we're expressing that we're the arbiters of the English language when what we want to say is that we're opting to use a specific dialect. Asking them to avoid using words that may have more than one spelling can potentially give the wrong impression.

Reply:
ah that's fair, i saw that google specified 'non-us' in their docs and my main point to get across was for ppl to avoid putting readers in a position where they have to write things using their regional english spelling. 

I'll commit this @matthewshaver thanks!